src/schema/infrastructure/disable-schema-integration.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Yoast\WP\SEO\Schema\Infrastructure;
+
+use Yoast\WP\SEO\Conditionals\Front_End_Conditional;
+use Yoast\WP\SEO\Conditionals\Schema_Disabled_Conditional;
+use Yoast\WP\SEO\Integrations\Integration_Interface;
+
+/**
+ * Handles disabling schema.
+ */
+class Disable_Schema_Integration implements Integration_Interface {
+
+	/**
+	 * Returns the conditionals based on which this loadable should be active.
+	 *
+	 * @return string[]
+	 */
+	public static function get_conditionals() {
+		return [
+			Front_End_Conditional::class,
```

Comment:
Come to think about it, this doesn't need the `Front_End_Conditional` because then it wouldnt work on the REST API.

Basically, if you have disabled schema via the option and visit a post's REST address (eg. `http://example.com/wp-json/wp/v2/posts/<POST_ID>` and search for `"schema"`, you will find the schema even though you shouldn't.

We can remove that conditional and do nothing else, or create a conditional that checks if we're in REST request, but I'm leaning towards the former.